disco_aws_automation/disco_alarm.py

```diff
@@ -10,6 +10,7 @@
 
 # Max batch size for alarm deletion http://goo.gl/vMQOrX
 DELETE_BATCH_SIZE = 100
+ENVIRONMENT_DELETE_SKIP_NAMESPACES = ['AWS/ES']
```

Comment:
We do not have a general config section in disco_alarms.ini, not entierly convinced we need one (yet). There is very little reason to change this value disco_vpc does not destroy ES clusters on destruction of environment, and there is no way of changing that behavior in config. As the option is not exposed there I see little point of exposing it here.